docs/configuration/configuration.md

```diff
@@ -1020,37 +1020,58 @@ Serverset data must be in the JSON format, the Thrift format is not currently su
 scrape targets from [Container Monitor](https://github.com/joyent/rfd/blob/master/rfd/0027/README.md)
 discovery endpoints.
 
-The following meta labels are available on targets during relabeling:
+One of the following `<triton_role>` types can be configured to discover targets:
+
+#### `container`
+
+The `container` role discovers one target per "virtual machine" owned by the `account`. These are smartos zones or lx/kvm/bhyve branded zones.
```

Comment:
Isn't it SmartOS?

Reply:
Yes, KVM also needs to be upper-case to stick with branding.

Reply:
Changed triton -> Triton where used in text, same for SmartOS and KVM.